cmd/bandd/cmd/multi_send.go

```diff
@@ -40,7 +41,7 @@ func MultiSendTxCmd() *cobra.Command {
 				inputCoins = inputCoins.Add(coins...)
 			}
 			msg := banktypes.NewMsgMultiSend(
-				[]banktypes.Input{banktypes.NewInput(sender, inputCoins)},
+				banktypes.NewInput(sender, inputCoins),
```

Comment:
Seems root command doesn't include `multi-send` anymore. should we remove this file?